config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
-  root 'root#index'
-end
\ No newline at end of file
+  root to: redirect('/api/all_greeting')
```

Comment:
- [ ] @bhushan354 Great work trying to fix the error shown by the previous reviewer, the error is occurring because your react app is not displaying anything, when I add the root route am getting the error below, which indicates an error on how you setup your react application, kindly consider following the tutorial again
![image](https://github.com/bhushan354/Rails-React-App-Webpack/assets/61408860/1c543da3-44a6-4d86-85d1-6966a6802467)